action.yml

```diff
@@ -0,0 +1,59 @@
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+inputs:
+  repo:
+    description: "The repo to run against"
+    required: false
+  instance_count:
+    description: "The number of instances to create"
+    required: false
+    default: '1'
+  gh_pat:
+    description: "The GHA PAT"
+    required: true
+  action:
+    description: 'Whether to start or stop. Options: "start", "stop"'
+    required: true
+  provider:
+    description: 'The cloud provider to use to provision a runner. Example: "aws"'
+    required: true
+  gh_runner_labels:
+    description: 'The comma seperated list of runner labels for GitHub when stopping'
+    required: false
```

Comment:
is this required when stopping? (when the command is `stop`?)

Reply:
It is.